### PR TITLE
AC Automated Test - LRAC-8487 | master

### PR DESCRIPTION
**Pull request by 91nilo**
https://issues.liferay.com/browse/LRAC-8487
https://issues.liferay.com/browse/LRAC-8887 (Refactor)

### CODE REVIEW

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverViewMembership.testcase [hunk 1]
```diff
@@ -386,12 +395,104 @@ definition {
...
-		// TODO LRAC-8487 PENDING IMPLEMENTATION
+				User.logoutAndLoginPG(
+					userLoginEmailAddress = "${username}@liferay.com",
+					userLoginFullName = "${fullName}");
+
+				Navigator.gotoSitePage(
+					pageName = "AC Page",
+					siteName = "Site Name");
+
+				Pause(locator1 = "10000");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverViewMembership.testcase:438]: Hi @91nilo, I think 10 seconds is a lot for this here, usually in these cases the other tests use 3 or 5 seconds. Could you check if yours would pass with this time?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverViewMembership.testcase [hunk 2]
```diff
@@ -386,12 +395,104 @@ definition {
...
+					userEmailAddress = "user${n}@liferay.com");
+			}
+		}
+
+		task ("Sync the Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("View the page as different users") {
+			for (var username : list "user1,user2") {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverViewMembership.testcase:427]: Hi @91nilo, you could create a `var` at the beginning of the test and call it in the `for` list.

**Comment by 91nilo** [comment]
Hi @ViniLopes20 i did the changes (using variable on the loops and a 3 seconds pause). 
If there is anything else i could do to improve the code, just let me know.
Thanks!

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverViewMembership.testcase [hunk 3]
```diff
@@ -386,12 +395,108 @@ definition {
...
 
-		// TODO LRAC-8487 PENDING IMPLEMENTATION
+		// AC Refactor: timing issue for membership card
+		// AC Refactor ticket: LRAC-8887
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		var userNumber = "1,2";
 
+		var userList = "user1,user2";
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverViewMembership.testcase:409]: Hi @91nilo, I think you could merge these lists

Comment (91nilo) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverViewMembership.testcase]: Done! Thanks

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by christianstokes** [comment]
ci:forward

**Pull request closed without merging**